mitodl/mit-learn
# add new learning resource drawer layout

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/5819

### Description (What does it do?)
This PR is a first pass at implementing the new wider learning resource drawer design. The structure and styles have been altered to match what's included in the design, minus the out of scope items listed in the issue. As part of this, the dropdown UI for selecting runs that changes the information section has been removed. Instead, all possible values across all runs are shown in a pipe separated list.

### Screenshots (if appropriate):
![image](https://github.com/user-attachments/assets/ae172d6d-94ea-49d1-9308-ff423b4f0771)
![image](https://github.com/user-attachments/assets/95e8188a-db19-414f-8e24-4488a8da0c7b)

### How can this be tested?
 - Spin up `mit-learn` on this branch
 - Ensure that your local database is sufficiently populated with enough learning resources for testing:
   - A variety of courses
   - Podcasts
   - Videos
 - Ensure that you have built a search index
 - Visit the search page at http://localhost:8062/search
 - Search for and click on learning resources to bring up the new drawer
 - Check all of the above resource type and ensure that the general structure and information presented matches what's expected and aligns with the provided designs
